boards/phytec/phyboard_polis/doc/index.rst

```diff
@@ -1,7 +1,7 @@
-.. _mimx8mm_phyboard_polis:
+.. _phyboard_polis:
 
-PhyBOARD Polis (NXP i.MX8M Mini)
-################################
+phyBOARD-Polis i.MX8M Mini
```

Comment:
One last request from me: please set the new names in the new `full_name` property of the respective `board.yml` files. Thanks!

Reply:
Ah nice, I will update the other boards soon too.